### PR TITLE
fix: added support for RTL to indeterminate animations and circular progress

### PR DESCRIPTION
**Pull request by SethDonohue**
# Description

Add directionSwitch to styles for progress component and an additional story to showcase a narrow container with progress bar
## Motivation & context

The Progress component did not handle Right To Left direction for animation on the line and circular versions, as well as static Right To Left progress on the circular version

https://github.com/microsoft/fast-dna/issues/2406

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-components-react-msft/src/progress/progress.stories.tsx [hunk 1]
```diff
@@ -2,8 +2,15 @@ import { storiesOf } from "@storybook/react";
 import React from "react";
 import { Progress } from "./";
 
+const progressStyle: any = {
```
Comment (chrisdholt) [inline packages/fast-components-react-msft/src/progress/progress.stories.tsx]: Is this actually *any*?

File: packages/fast-components-styles-msft/src/progress/index.ts [hunk 2]
```diff
@@ -139,59 +140,59 @@ const styles: ComponentStyles<ProgressClassNameContract, DesignSystem> = {
     },
     progress_dot__1: {
         width: "40%",
-        transform: "translateX(-100%)",
+        // transform: directionSwitch("translateX(-100%)", "translateX(100%)"),
```
Comment (chrisdholt) [inline packages/fast-components-styles-msft/src/progress/index.ts]: Should this be commented out? (Same with line 148

Comment (SethDonohue) [reply packages/fast-components-styles-msft/src/progress/index.ts]: Obviously not, sorry!

File: packages/fast-components-react-msft/src/progress/progress.stories.tsx (hunk 1 above)
Comment (SethDonohue) [reply packages/fast-components-react-msft/src/progress/progress.stories.tsx]: Importing these from `@microsoft/fast-components-class-name-contracts-msft` requires that I we have 
```
progress_circularSVG__control
progress_circularSVG__container
progress_circularSVG__page
```

Shouldn't all of the classnames in the contract be optional?

**Pull request merged**